layouts/docs/baseof.html

```diff
@@ -5,28 +5,28 @@
   </head>
   <body class="td-{{ .Kind }}">
     <header>
-      {{ partial "navbar-docs.html" . }}
+      {{ partial "navbar.html" . }}
```

Comment:
Do we use the navbar-docs.html partial anywhere else?  If not suggest deleting that file.